pkg/cassandra/gocql/testutils/udt_test.go

```diff
@@ -79,3 +80,7 @@ func TestUDTTestCase(t *testing.T) {
 
 	testCase.Run(t)
 }
+
+func TestMain(m *testing.M) {
+	goleak.VerifyTestMain(m)
```

Comment:
don't use this directly call `testutils.VerifyGoLeaks`  form `pkg/testutils`

Reply:
thanks @akagami-harsh ,looking into it